client/FxAccountClient.js

```diff
@@ -1134,6 +1134,49 @@ define([
   };
 
   /**
+   * Send an sms.
+   *
+   * @method sendSms
+   * @param {String} sessionToken SessionToken obtained from signIn
+   * @param {String} phoneNumber Phone number sms will be sent to
+   * @param {String} messageId Corresponding message id that will be sent
+   * @param {Object} [options={}] Options
+   *   @param {String} [options.lang] lang Language that sms will be sent in
+   *   @param {Object} [options.metricsContext={}] Metrics context metadata
+   *     @param {String} options.metricsContext.flowId identifier for the current event flow
+   *     @param {Number} options.metricsContext.flowBeginTime flow.begin event time
+   */
+  FxAccountClient.prototype.sendSms = function (sessionToken, phoneNumber, messageId, options) {
+
+    required(sessionToken, 'sessionToken');
+    required(phoneNumber, 'phoneNumber');
+    required(messageId, 'messageId');
+
+    var data = {
+      phoneNumber: phoneNumber,
+      messageId: messageId
+    };
+    var requestOpts = {};
+
+    if (options) {
+      if (options.lang) {
+        requestOpts.headers = {
+          'Accept-Language': options.lang
+        };
+      }
+
+      if (options.metricsContext) {
+        data.metricsContext = metricsContext.marshall(options.metricsContext);
+      }
+    }
+
+    return hawkCredentials(sessionToken, 'sessionToken',  HKDF_SIZE)
+      .then(function(creds) {
+        return this.request.send('/sms', 'POST', creds, data, requestOpts);
```

Comment:
`this` inside the function here will be undefined, so you'll need to save a reference to `this.request`. I'm a bit surprised the tests didn't catch this. I only caught it while doing an integration test with the built resources.